backend/src/models/Poll.test.ts

```diff
@@ -141,9 +149,12 @@ describe('Poll', () => {
 
     describe('loadFromDatabase', () => {
         test('Load poll from database', async () => {
-            prismaMock.poll.findFirst.mockResolvedValue(dummyDatabaseData);
+            // This should be changed, just set it as any in hurry - Joonas Hiltunen 02.10.2022
+            prismaMock.poll.findFirst.mockResolvedValue(
+                dummyDatabaseData as any
```

Comment:
I fixed this ´any´ as a part of the previous refactor PR, it should be updated from there.